nav2_system_tests/src/costmap_filters/CMakeLists.txt

```diff
@@ -5,10 +5,9 @@ ament_add_test(test_keepout_filter
   TIMEOUT 180
   ENV
     TEST_DIR=${CMAKE_CURRENT_SOURCE_DIR}
-    TEST_MAP=${PROJECT_SOURCE_DIR}/maps/map_circular.yaml
```

Comment:
Removed map across all of these? Isn't that necessary?

Reply:
Its good to keep these tests parameterized

Reply:
I replaced the variables TEST_MAP and TEST_WORLD with `get_package_share_directory ('aws_robomaker_small_warehouse_world')` because I assume _aws_robomaker_small_warehouse_world_ package is installed from binaries.